.github/workflows/artifacts.yaml

```diff
@@ -373,3 +382,126 @@ jobs:
         uses: github/codeql-action/upload-sarif@407ffafae6a767df3e0230c3df91b6443ae8df75 # v2.22.8
         with:
           sarif_file: trivy-results.sarif
+
+  crd-subchart:
+    if: github.repository == 'kube-logging/logging-operator'
+    name: CRD subchart
+    runs-on: ubuntu-latest
+  
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+      security-events: write
+  
+    outputs:
+      name: ${{ steps.oci-chart-name.outputs.value }}
+      tag: ${{ steps.version.outputs.value }}
+      package: ${{ steps.build.outputs.package }}
+  
+    env:
+      subchartPath: logging-operator/charts/crds
+      subchartName: crds
```

Comment:
[Nit]
I would rename it to `logging-operator-crds`, would it break anything?

Reply:
If you check the [uploaded artifacts](https://github.com/kube-logging/logging-operator/pkgs/container/helm-charts%2Flogging-operator%2Fcrds), it uses a full-path that contains the repo-name. `kube-logging/helm-charts/logging-operator/crds`